<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>历史今日查询</string>
  </property>
  <widget class="QComboBox" name="monBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>月：</string>
   </property>
  </widget>
  <widget class="QLabel" name="mon_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>日：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dayBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="bt">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="showText">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>450</width>
     <height>500</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bt2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>选中事件详细信息查询</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>450</width>
     <height>500</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>事件</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
